test/test26.cxx

```diff
@@ -33,18 +33,14 @@ std::map<int, int> update_years(connection_base &C)
   std::map<int, int> conversions;
   work tx{C};
 
-  // First select all different years occurring in the table.
-  result R{tx.exec("SELECT year FROM pqxxevents")};
-
   // Note all different years currently occurring in the table, writing them
   // and their correct mappings to m_conversions
-  for (auto const &r : R)
+  for (auto const [y] :
```

Comment:
Why bother with structured bindings if you're only pulling out a single member object?

Reply:
Is there a way to do it without  structured bindings, besides adding an alternative function to call?